src/panels/config/logs/error-log-card.ts

```diff
@@ -19,21 +19,34 @@ class ErrorLogCard extends LitElement {
 
   protected render(): TemplateResult {
     return html`
-      <p class="error-log-intro">
+      <div class="error-log-intro">
         ${this._errorLog
           ? html`
-              <ha-icon-button
-                icon="hass:refresh"
-                @click=${this._refreshErrorLog}
-              ></ha-icon-button>
+              <ha-card>
+                <ha-icon-button
+                  icon="hass:refresh"
+                  @click=${this._refreshErrorLog}
+                ></ha-icon-button>
+                <div class="card-content error-log">
+                  ${this._errorLog.split("\n").map((entry) => {
```

Comment:
Don't do this in render, as we will then do it on every update of hass, move it to `_refreshErrorLog`